pyartcd/pyartcd/pipelines/update_golang.py

```
@@ -1163,6 +1174,324 @@ async def _rebase_and_build_konflux(self, el_v, go_version, go_nvr: str):
+        quay_auth_file = self._get_required_env('QUAY_AUTH_FILE')
+        kubeconfig = self._get_required_env('KUBECONFIG')
+        qci_user = self._get_required_env('QCI_USER')
+        qci_password = self._get_required_env('QCI_PASSWORD')
+
+        return RegistryConfig(
+            source_files=[quay_auth_file],
+            kubeconfig=kubeconfig,
+            registries=[
+                REGISTRY_CI_OPENSHIFT,
+                REGISTRY_QUAY_OCP_RELEASE_DEV,
+                KONFLUX_DEFAULT_IMAGE_REPO,
```

Ruff (0.16.2)

[error] Undefined name `KONFLUX_DEFAULT_IMAGE_REPO`

(F821)